addons/stopwatch/stopwatch.lua

```diff
@@ -0,0 +1,87 @@
+_addon.name = 'stopwatch'
+_addon.author = 'Patrick Finnigan (Puhfyn@Ragnarok)'
+_addon.version = '1.0'
+_addon.commands = {'sw', 'stopwatch'}
+
+require('logger')
+config = require('config')
+texts = require('texts')
+ 
+defaults = {}
+defaults.pos = {}
+defaults.pos.x = 450
+defaults.pos.y = 0
+defaults.text = {}
+defaults.text.font = 'Arial'
+defaults.text.size = 12
+ 
+settings = config.load(defaults)
+times = texts.new(settings)
+startTime = os.time()
+isActive = false
+cumulativeSeconds = 0
+
+
+windower.register_event('prerender', function()
+    local info = windower.ffxi.get_info()
+    if not info.logged_in then
+        times:hide()
+        return
+    end
+
+    seconds = cumulativeSeconds
+    if isActive == true then
+        seconds = seconds + os.time() - startTime
```

Comment:
Good to know. Is this because os.difftime is more cross platform? Looks like subtraction is equivalent on Windows: http://www.lua.org/manual/5.1/manual.html#pdf-os.difftime